cmd/cli/main.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 func main() {
-    app := &cli.App{
+	app := &cli.App{
```

Comment:
I'm curious, did your editor make these space/tab changes through the files? Or was it the result of go fmt? Because if it's the former then it should be the latter.

Reply:
@richardhallett it was done by the standard go tooling. i noticed that it was formatting the files i worked on, on save (this is how i've set up my editor). so i ran `go fmt ./...` on the project to format it such that it adheres to the go standards.